docs/ref-configuration.md

```diff
@@ -22,7 +22,7 @@ The controllers can be started with these environment variables:
 
 In cluster configuration for the agent and fleet manager. Changing these can lead to full re-deployments.
 
-The config [struct](https://github.com/rancher/fleet/blob/main/pkg/config/config.go#L40-L52) is used in both config maps:
+The config [struct](https://github.com/rancher/fleet/blob/main/internal/config/config.go#L40-L52) is used in both config maps:
```

Comment:
The config struct has moved a bit since this was written ;)
How about we just point to its beginning, to make this document more future-proof?
```suggestion
The config [struct](https://github.com/rancher/fleet/blob/main/internal/config/config.go#L57) is used in both config maps:
```

Reply:
Thanks for suggestion.